AirSkyBoat/AirSkyBoat
# Clamped spellcasting times

**_I affirm:_**


## Please enter a player-facing description
- Fixed an issue that prevented spell casting when it was reduced below 0. (sarca571ca)


## What does this pull request do? (Please be technical)
Adds a clamp to spell casting to prevent it from going below 1 sec. Type was changed from <unit32> to <int32> so it could handle negative values, otherwise it would increase cast time(in my testing at least).


## Steps to test these changes
Use "!setmod 1078 3" while targeting yourself. Then summon a elemental spirit and see that cast time cannot go below 1 second.


## Special Deployment Considerations
This is a core change so a rebuild is required for changes to take effect.
